dts/bindings/can/atmel,sam-can.yaml

```diff
@@ -10,14 +10,17 @@ properties:
   reg:
     required: true
     description: |
-      2 reg blocks needed; Register block for the MCAN configuration registers;
-      Register block for the DMA Base Address
+      2 or 3 reg blocks needed;
```

Comment:
The devicetree binding update must be extract from this commit. The commit must be the first in the series.